pyfolio/timeseries.py

```diff
@@ -752,6 +752,131 @@ def perf_stats(returns, factor_returns=None):
     return stats
 
 
+def perf_stats_bootstrap(returns, factor_returns=None, return_stats=True):
+    """Calculates various bootstrapped performance metrics of a strategy.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+         - See full explanation in tears.create_full_tear_sheet.
+    factor_returns : pd.Series (optional)
+        Daily noncumulative returns of the benchmark.
+         - This is in the same style as returns.
+        If None, do not compute alpha, beta, and information ratio.
+    return_stats : boolean (optional)
+        If True, returns a DataFrame of mean, 5 and 95 percentiles for
+        each perf metric.
+        If False, returns a DataFrame with the bootstrap samples for
+        each perf metric.
+
+    Returns
+    -------
+    pd.DataFrame
+        if return_stats is True:
+        - Distributional statistics of bootstrapped sampling
+        distribution of performance metrics.
+        if return_stats is False:
+        - Bootstrap samples for each performance metric.
+    """
+    bootstrap_values = OrderedDict()
+
+    for stat_func in SIMPLE_STAT_FUNCS:
+        stat_name = stat_func.__name__
+        bootstrap_values[stat_name] = calc_bootstrap(stat_func,
+                                                     returns)
+
+    if factor_returns is not None:
+        for stat_func in FACTOR_STAT_FUNCS:
+            stat_name = stat_func.__name__
+            bootstrap_values[stat_name] = calc_bootstrap(stat_func,
+                                                         returns,
+                                                         factor_returns)
+    bootstrap_values = pd.DataFrame(bootstrap_values)
+
+    if return_stats:
+        stats = bootstrap_values.apply(calc_distribution_stats)
+        return stats.T[['mean', '5%', '95%']]
```

Comment:
Do we want to include median and/or IQR here? Would allow the user a way to asses the normality of the bootstrap results.